foundation/browser/README.md

```diff
@@ -0,0 +1,82 @@
+<p align="center">
+  <a href="https://github.com/ForgeRock/ping-android-sdk">
+    <img src="https://www.pingidentity.com/content/dam/picr/nav/Ping-Logo-2.svg" alt="Logo">
+  </a>
+  <hr/>
+</p>
+
+# Ping Browser
+
+## Overview
+
+Ping Browser is a library that allows you to perform an in-app browser action, and receive data from the browser
+redirect.
+
+By default, the library uses the `CustomTabsIntent` to launch the browser, and it provides a callback to receive the
+data from the browser redirect. However, you can customize the browser launcher to use a different browser or customize
+the custom tabs.
+
+```mermaid
+sequenceDiagram
+    participant App
+    participant BrowserLauncher
+    participant Browser
+    participant Server
+    App ->> BrowserLauncher: launch(url)
+    BrowserLauncher ->> Browser: Launch with URL
+    Browser ->> Server: HTTP request
+    Server ->> Browser: Response with redirect URI
+    Browser ->> BrowserLauncher: URI
+    BrowserLauncher ->> App: URI
+    Note over App, Browser: Complete redirect flow
+```
+
+## Add dependency to your project
+
+```kotlin
+dependencies {
+    implementation(project(":foundation:browser"))
```

Comment:
This suggests importing local module, but customers will be using maven... so maybe this can be made more clear...